treeherder/etl/artifact.py

```diff
@@ -161,7 +162,7 @@ def serialize_artifact_json_blobs(artifacts):
         blob = artifact['blob']
         if (artifact['type'].lower() == 'json' and
                 not isinstance(blob, str) and
-                not isinstance(blob, unicode)):
+                not isinstance(blob, six.text_type)):
```

Comment:
Sorry missed this before. I think this could be changed to [string_types](https://pythonhosted.org/six/#six.string_types) and the `str` check prior removed as well? (Since previously the conditional here was really checking for [basestring](https://docs.python.org/2/library/functions.html#basestring))

Reply:
Fixed in cab7f575